src/scicookie/{{cookiecutter.project_slug}}/build-system/flit-pyproject.toml

```diff
@@ -38,7 +38,7 @@ classifiers = [
 dynamic = ["version"]
 requires-python = ">=3.8.1"
 dependencies = [
-    "urllib3 =< 2",
+    "urllib3 <= 2",
```

Comment:
@xmnlab Sorry for this… just for urllib3 when run `flit install` show a message about it:
`
  Attempting uninstall: urllib3
    Found existing installation: urllib3 2.0.3
    Uninstalling urllib3-2.0.3:
      Successfully uninstalled urllib3-2.0.3
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
cattrs 22.2.0 requires exceptiongroup; python_version < "3.11", which is not installed.
`
Do you think that we should remove it?

Reply:
thanks @Anavelyz for caching that. you're right, we need to pin that just for poetry :) (because it introduces a bunch of issues with poetry)

I will remove that. thanks!